app/models/notification.rb

```diff
@@ -139,6 +139,7 @@ def self.types
                         assigned: 34,
                         question_answer_user_commented: 35, # Used by https://github.com/discourse/discourse-question-answer
                         watching_category_or_tag: 36,
+                        new_features: 37,
```

Comment:
Note for me and/or @lis2, this will need support added to DiscourseHub.

Reply:
@pmusaraj FYI `watching_category_or_tag: 36` was added recently too. does that need to be supported explicitly as well?

Reply:
Yes, @mcwumbly, any new notification type needs to be added explicitly to DiscourseHub. The notification will show up as "Unmapped type: ID" in the app's notifications screen in the meantime.